rtcrtpreceiver.go

```diff
@@ -117,6 +120,7 @@ func (r *RTCRtpReceiver) Receive(parameters RTCRtpReceiveParameters) chan bool {
 			select {
 			case r.rtcpOut <- rtcpPacket:
 			default:
+				break
```

Comment:
I don't believe we need/want this `break`

This exists so when giving packets to the user we discard them if they aren't listening/running behind. We should probably make this a switch, but currently it is lossy so we can always be live.